src/core/src/collection.rs

```diff
@@ -409,8 +409,8 @@ mod test {
         // no sigs should remain
         assert_eq!(cl.len(), 6);
         for (_idx, rec) in cl.iter() {
-            // need to pass select again here so we actually downsample
-            let this_sig = cl.sig_from_record(rec).unwrap().select(&selection).unwrap();
+            eprintln!("record scaled is: {}", rec.scaled());
```

Comment:
We have a couple of `eprintln!` sneaking into the codebase, it's preferable to use `info!` or `debug!` (or even `trace!`?) from the `log` crate

```suggestion
            debug!("record scaled is: {}", rec.scaled());
```

Reply:
I mean in the tests it's ok, right?

Reply:
oh, good point! it will likely also break without the right import, I don't think it is happening here.

even so, in tests `dbg!` is fine too, because it shows the line in the code where it is called, and `eprintln!` doesn't